### PR TITLE
Adding Status-React Quickstart & images.

### PR DESCRIPTION
**Pull request by rbnpercy**

### CODE REVIEW

File: source/guides/status_react_quickstart.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+The goal of this tutorial is to get someone new to Status-react to the point where they can evaluate expressions in-line in their editor. Inline evaluation is one of the things that makes Clojure both fun and powerful.
+
+What follows is the definitive account of how to get to that “aha” moment, specifically for the **iOS dev build of Status-react**.  With this built, you can start contributing as soon as you wish!  
+
+**If you want to contribute to a different build (i.e. the desktop client) you can read [*this guide*](https://status.im/guides/desktop_inline_eval.html).**
+
+
+
+# Installing Clojure &amp; CLJS
```
Comment (jakubgs) [inline source/guides/status_react_quickstart.md]: Why is this necessary? Clojure is already installed via Nix, all you need is `make shell TARGET=android` (or some other target).
https://github.com/status-im/status-react/blob/79f2eed181a0c33d981dbbba66c5717aba3df170/shell.nix#L29-L40

File: source/guides/status_react_quickstart.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
+---
+id: status_react_quickstart
+title: Status-React Quickstart Guide
```
Comment (jakubgs) [inline source/guides/status_react_quickstart.md]: Why are we putting this in `source/guides` rather than `source/build_status`?

**Comment by jakubgs** [comment]
Also, a description and purpose of the changes in the PR would be good.

**Comment by rbnpercy** [comment]
> Also, a description and purpose of the changes in the PR would be good.

I understand that for more complex pull requests, but the title really says it all in this case - fairly self explanatory is it not?  I'll add descriptions in any future PRs
